feat(content): Avgi Culture: Fencing (#11153)
endless-sky/endless-sky: data/avgi/avgi culture conversations.txt

			`The spaceport is lit up by flashing yellow lights as a robotic voice proclaims its message. "Command. Radiation danger, seek prompt shelter. Command. Radiation danger, seek prompt shelter."`
			`	The crowd has only barely had time to panic before the voice is cut off with a squeal of static. "Abashment. Apologies, false alert. Mindaro dropped his drink on the console, there is no radiation danger."`
				decline

mission "Avgi Culture: Fencing"
	minor
	invisible
	source
		government "Avgi (Dissonance)"
	to offer
		not "avgi: lost in twilight"
		random < 3
	on offer
		conversation

> 			`While walking through the spaceport on <planet>, you spot a pair of Avgi moving somewhere with a purpose holding what look like long, thin rapiers.`

Reviewer: ```suggestion
			`While walking through the spaceport on <planet>, you spot a pair of Avgi moving toward a large building. They are holding what look like long, thin rapiers.`
```
This sentence is a bit run-on otherwise.

			choice
				`	(Approach them.)`
				`	(Ignore them.)`
					goto ignore

> 			`	One of the Avgi speaks up, "Invitation. Human, are you perhaps, interested in witnessing the ancient Avgi sport of fencing?"`

Author: Added more about their behavior at the beginning and their appearances

> 			`	One of the Avgi speaks up, "Invitation. Human, are you perhaps, interested in witnessing the ancient Avgi sport of fencing, question?"`

Reviewer: ```suggestion
			`	One of the Avgi speaks up, "Invitation. Human, are you perhaps interested in witnessing the ancient Avgi sport of fencing, question?"`
```

			choice
				`	"Fencing? Like the human sport?"`
				`	"Sure, I'm willing to watch."`
					goto accept
				`	"I'm good, thanks."`
					goto ignore

> 			`	The other chimes in, "Assurance. Avgi have surely partaken in this most daring dance far longer than humans. I implore you to attend."`

Author: Yes, but would these particular Avgi care about details like that or would they have large egos?

> 			`	The other chimes in, "Assurance. Avgi have surely partaken in this most daring dance far longer than humans. I implore you to attend."`

Author: Fair. And considering the rest of Avgi conversations related to their human counterparts, they probably wouldn't double down on an incorrect statement. I'll work out a better sentence.

> 			`	The first Avgi continues, "Reinforcement. Considering our... differences in agility, this will surely be a spectacle to you, compared to human sports that is."`

Author: I also struggled with that one, open to suggestions

> 			`	The first Avgi continues, "Reinforcement. Considering our... differences in agility, this will surely be a spectacle to you, compared to human sports that is."`

Author: Loy suggested Addenum, I think it works better

> 			`	The first Avgi continues, "Reinforcement. Considering our... differences in agility, this will surely be a spectacle to you, compared to human sports that is."`

Reviewer: How about this:
```suggestion
			`	The first Avgi continues, "Addendum. Considering our... differences in agility, this will surely be a spectacle to you, compared to human sports that is."`
```

			choice
				`	"Sure, sounds like fun!"`
				`	"I'll pass, thanks."`
					goto ignore
			label accept
			branch language
				has "language: Avgi (Written)"

> 			`	A short walk later, you find yourself in a large open-air room, similar to that of a gymnasium albeit much taller. Rows of skate park-like rails make up what appear to be bleachers the whole way around this cylindrical room, packed to the brim with Avgi perched atop them. Many are holding up signs with colorful lettering that appear to be in favor of one participant or the other.`

Reviewer: ```suggestion
			`	A short walk later, you find yourself in a large open-air room, similar to a gymnasium but much taller. Rows of rails that look like they came from a skate park make up bleachers the whole way around the cylindrical area, packed to the brim with perched Avgi. Many are holding up signs with colorful lettering that appear to be in favor of one participant or the other.`
```
"but" goes with the flow better here, even though it means the same thing.

> 					goto start

Reviewer: ```suggestion
				goto start
```

			label language

> 			`	A short walk later, you find yourself in a large open-air room, similar to that of a gymnasium albeit much taller. Rows of skate park-like rails make up what appear to be bleachers the whole way around this cylindrical room, packed to the brim with Avgi perched atop them. Many are holding up signs that appear to be in favor of one participant or the other. One sign in particular states "ZOMP TO STOMP THEM LIKE THE CONSONANCE!"`

Author: done

> 			`	A short walk later, you find yourself in a large open-air room, similar to that of a gymnasium albeit much taller. Rows of skate park-like rails make up what appear to be bleachers the whole way around this cylindrical room, packed to the brim with Avgi perched atop them. Many are holding up signs that appear to be in favor of one participant or the other. One sign in particular states "ZOMP TO STOMP THEM LIKE THE CONSONANCE!"`

Reviewer: ```suggestion
			`	A short walk later, you find yourself in a large open-air room, similar to a gymnasium but much taller. Rows of rails that look like they came from a skate park make up bleachers the whole way around the cylindrical area, packed to the brim with perched Avgi. Many are holding up signs with colorful lettering that appear to be in favor of one participant or the other. One sign in particular states "ZOMP TO STOMP THEM LIKE THE CONSONANCE!"`
```

			label start

> 			`	Soon after, the two Avgi you approached earlier walk out of a small door on the bottom level. The Avgi in the arena erupt with a flurry of wing flapping in what seems to be a loud cheer. Your translator picks up "Excitement" and "Anticipation" among other moods dozens of times. The fencers' wings appear to be kitted out with thin copper wires, and they're carrying the same rapiers you saw them with before. They hop up onto small pedestals located on each edge of the arena.`

Author: Consider it done

> 			`	Soon after, the two Avgi you approached earlier walk out of a small door on the bottom level. The Avgi in the arena erupt with a flurry of wing flapping in what seems to be a loud cheer. Your translator picks up "Excitement" and "Anticipation" among other moods dozens of times. The fencers' wings appear to be kitted out with thin copper wires, and they're carrying the same rapiers you saw them with before. They hop up onto small pedestals located on each edge of the arena.`

Author: No, it's like the suits human fencers wear. If the sword touches their wings, it sends the electrical contact ending the fight.

> 			`	Soon after, the two Avgi you approached earlier walk out of a small door on the bottom level. The Avgi in the arena erupt with a flurry of wing flapping in what seems to be a loud cheer. Your translator picks up "Excitement" and "Anticipation" among other moods dozens of times. The fencers' wings appear to be kitted out with thin copper wires, and they're carrying the same rapiers you saw them with before. They hop up onto small pedestals located on each edge of the arena.`

Reviewer: ```suggestion
			`	Soon after, the two Avgi you met earlier walk out of a small door on the bottom level. The spectators erupt with a flurry of wing flapping in what seems to be a loud cheer; your translator picks up "Excitement" and "Anticipation" among other moods dozens of times. The fencers' wings appear to be kitted out with thin copper wires, and they're carrying the same rapiers you saw them with before. They hop up onto small pedestals located on each edge of the arena.`
```

Author: This one got conflicted in the poorly timed commit but all the relevant changes are in

> 			`	A booming voice is heard over the many speakers present in the room. "Proclamation. Here we have our reigning champion, Razmatazz, and upstart challenger Zomp! Are you ready to witness this enthralling battle between these two paragons of the blade, question?"`

Reviewer: ```suggestion
			`	A voice booms over the many speakers present in the room. "Proclamation. Here we have our reigning champion, Razmatazz, and the upstart challenger Zomp! Are you ready to witness this enthralling battle between these two paragons of the blade, question?"`
```

			`	The arena once more hums to life with the flutter of wings. "Anticipation. Then let the duel... begin."`

> 			`	With the announcers word, the duelists leap into the air with incredible speed, circling around each other and clashing blades. The two spiral upwards into the far upper areas of the arena, and quickly drop back down, darting in and out of each others' reach multiple times per second. Less than a minute later, you hear a loud ding, and one of the duelists falls to the floor with their wing collapsed. The crowd, who hadn't stopped buzzing the entire duel, erupts with even louder wing fluttering.`

Reviewer: ```suggestion
			`	The moment the announcer finishes, the duelists leap into the air with incredible speed, circling around each other and clashing blades. The two spiral upwards into the far upper areas of the arena, and quickly drop back down, darting in and out of each others' reach multiple times per second. Less than a minute later, you hear a loud ding, and one of the duelists falls to the floor with their wing collapsed. The crowd, who hadn't stopped buzzing the entire duel, erupts with even louder wing fluttering.`
```

Author: Not a fan of this change personally, the word in question is 'begin', so it makes sense they take flight with that word.

Reviewer: ```suggestion
			`	At the announcer's word, the duelists leap into the air with incredible speed, circling around each other and clashing blades. The two spiral upwards into the far upper areas of the arena, and quickly drop back down, darting in and out of each others' reach multiple times per second. Less than a minute later, you hear a loud ding, and one of the duelists falls to the floor with their wing collapsed. The crowd, who hadn't stopped buzzing the entire duel, erupts with even louder wing fluttering.`
```
How about this?

> 			`	"Astonishment. Zomp wins round one." the announcer states, seeming surprised at the result. "Repose. Shall we begin round two?"`

Reviewer: ```suggestion
			`	"Astonishment. Zomp wins round one." the announcer states. "Repose. Shall we begin round two?"`
```
You don't need to state the announcer's emotions; they just said them.

			`	Once more the contestants line up and spring into the air in a duel of eye-watering speed and agility. And once more, Zomp wins to the adoration of the crowd.`

> 			`	After the hubbub and celebration die down, you find no issue leaving the crowded arena as most Avgi simply fly over your head.`

Author: done

> 			`	After the hubbub and celebration die down, you find no issue leaving the crowded arena as most Avgi simply fly over your head.`

Reviewer: ```suggestion
			`	After the hubbub and celebration die down, you find no issue leaving the crowded arena - most Avgi simply fly over your head.`
```

				decline
			label ignore

> 			`	You leave the two to walk away, continuing on your own business.`

Reviewer: ```suggestion
			`	You let the two fly away, continuing on your own business.`
```

			`The spaceport is lit up by flashing yellow lights as a robotic voice proclaims its message. "Command. Radiation danger, seek prompt shelter. Command. Radiation danger, seek prompt shelter."`
			`	The crowd has only barely had time to panic before the voice is cut off with a squeal of static. "Abashment. Apologies, false alert. Mindaro dropped his drink on the console, there is no radiation danger."`
				decline

mission "Avgi Culture: Fencing"
	minor
	invisible
	source
		government "Avgi (Dissonance)"
	to offer
		not "avgi: lost in twilight"
		random < 15
	on offer
		conversation
			`While walking through the spaceport on <planet>, you spot a pair of Avgi moving toward a large building. They are holding what look like long, thin rapiers.`
			choice
				`	(Approach them.)`
				`	(Ignore them.)`
					goto ignore

> 			`	You approach the two vibrantly colored Avgi, who look you up and down as you walk over.`
> 			`	One of the Avgi, colored a muted teal, speaks up. "Invitation. Human, are you perhaps interested in witnessing the ancient Avgi sport of fencing, question?"`

Reviewer: ```suggestion
			`	As you walk over to the two vibrantly colored Avgi, they glance in your direction. One of them, colored a muted teal, speaks up. "Invitation. Human, are you perhaps interested in witnessing the ancient Avgi sport of fencing, question?"`
```

> 			`	One of the Avgi, colored a muted teal, speaks up saying "Invitation. Human, are you perhaps, interested in witnessing the ancient Avgi sport of fencing, question?"`

Reviewer: ```suggestion
			`	One of the Avgi, colored a muted teal, speaks up. "Invitation. Human, are you perhaps interested in witnessing the ancient Avgi sport of fencing, question?"`
```

			choice
				`	"Fencing? Like the human sport?"`
				`	"Sure, I'm willing to watch."`
					goto accept
				`	"I'm good, thanks."`
					goto ignore

> 			`	The other, colored a bright blue and purple sheen chimes in, "Assurance. Avgi have surely partaken in this most daring dance far longer than humans. I implore you to attend."`

Reviewer: ```suggestion
			`	The other, colored a bright blue and purple sheen, chimes in. "Assurance. Avgi have surely partaken in this most daring dance far longer than humans. I implore you to attend."`
```

> 			`	The first Avgi continues, "Addendum. Considering our... differences in agility, this will surely be a spectacle to you, compared to the human sport that is."`

Reviewer: ```suggestion
			`	The first Avgi continues, "Addendum. Considering our... differences in agility, this will surely be a spectacle to you, compared to the human sport, that is."`
```

Author: Feels a bit run-on now that I look at it. Perhaps terminate the sentence after "spectacle to you" and keep the comma in the next sentence?

Reviewer: ```suggestion
			`	The first Avgi continues, "Addendum. Considering our... differences in agility, this will surely be a spectacle to you. Compared to the human sport, that is."`
```

			choice
				`	"Sure, sounds like fun!"`
				`	"I'll pass, thanks."`
					goto ignore
			label accept
			branch language
				has "language: Avgi (Written)"
			`	A short walk later, you find yourself in a large open-air room, similar to a gymnasium but much taller. Rows of rails that look like they came from a skate park make up bleachers the whole way around the cylindrical area, packed to the brim with perched Avgi. Many are holding up signs with colorful lettering that appear to be in favor of one participant or the other.`
				goto start
			label language
			`	A short walk later, you find yourself in a large open-air room, similar to a gymnasium but much taller. Rows of rails that look like they came from a skate park make up bleachers the whole way around the cylindrical area, packed to the brim with perched Avgi. Many are holding up signs with colorful lettering that appear to be in favor of one participant or the other. One sign in particular states "ZOMP TO STOMP THEM LIKE THE ASSEMBLY!"`
			label start
			action

> 				set "avgi: watch zomp win fencing"

Reviewer: ```suggestion
				set "avgi: watched zomp win at fencing"
```

> 				log "People" "Zomp" `Zomp is a muted teal Avgi who defeated the reigning champion Razmatazz in a fencing match.`

Reviewer: Unless Zomp will appear elsewhere, this should be changed to
```suggestion
				log "Minor People" "Zomp" `Zomp is a muted teal Avgi who defeated the reigning champion Razmatazz in a fencing match.`
```

				log `Watched a fencing match between two Avgi onboard a Dissonance station with an extremely rowdy and excitable crowd. Witnessed an upset as an Avgi named Zomp defeated the reigning champion.`

> 			`	Soon after, the familiar muted teal and glistening purple Avgi walk out of a small door on the bottom level. The Avgi in the arena erupt with a flurry of wing flapping in what seems to be a loud cheer. Your translator picks up "Excitement" and "Anticipation" among other moods dozens of times. The fencers' wings appear to be kitted out with thin copper wires, and they're carrying the same rapiers you saw them with before. They hop up onto small pedestals located on each edge of the arena.`

Reviewer: ```suggestion
			`	Soon after, the familiar muted teal and glistening purple Avgi walk out of a small door on the bottom level. The spectators in the arena erupt with a flurry of wing flapping in what seems to be a loud cheer; your translator picks up "Excitement" and "Anticipation", among other moods, dozens of times. The fencers' wings appear to be kitted out with thin copper wires, and they're carrying the same rapiers you saw them with before. They hop up onto small pedestals located on each edge of the arena.`
```
Apparently GitHub autoresolved this when you made changes to the first sentence.

			`	A voice booms over the many speakers present in the room. "Proclamation. Here we have our reigning champion, Razmatazz, and the upstart challenger Zomp! Are you ready to witness this enthralling battle between these two paragons of the blade, question?"`
			`	The arena once more hums to life with the flutter of wings. "Anticipation. Then let the duel... begin."`
			`	At the announcer's word, the duelists leap into the air with incredible speed, circling around each other and clashing blades. The two spiral upwards into the far upper areas of the arena, and quickly drop back down, darting in and out of each others' reach multiple times per second. Less than a minute later, you hear a loud ding, and one of the duelists falls to the floor with their wing collapsed. The crowd, who hadn't stopped buzzing the entire duel, erupts with even louder wing fluttering.`

> 			`	"Astonishment. Zomp wins round one." the announcer states. "Repose. Shall we begin round two?"`

Reviewer: ```suggestion
			`	"Astonishment. Zomp wins round one," the announcer states. "Repose. Shall we begin round two?"`
```

			`	Once more the contestants line up and spring into the air in a duel of eye-watering speed and agility. And once more, Zomp wins to the thunderous applause of the crowd.`

> 			`	"Bewilderment. It seems as though Zomp has defeated our reigning champion!" The announcer proclaims.`

Reviewer: ```suggestion
			`	"Bewilderment. It seems as though Zomp has defeated our reigning champion!" the announcer proclaims.`
```

> 			`	"Hysteria. I knew he would win! I'm up 10:1 on this bet!" A nearby Avgi says with exuberance as they buzz around you wildly, holding various pink papers in their many arms.`

Reviewer: ```suggestion
			`	"Hysteria. I knew he would win! I'm up 10:1 on this bet!" a nearby Avgi screams with exuberance as they buzz around you wildly, holding various pink papers in their many arms.`
```

Author: I don't know if I'd describe it as a 'scream', more of frenzied shout. Not sure what word works there.

Author: Yeah, I think "says with exuberance" sums up the tone I had in my head for that moment, another word could work but it definitely isn't a "scream with exuberance."

Reviewer: updated the suggestion

Author: "10 to 1" is how it would be said, so I've updated it as such

> 			`	"Hysteria. I knew he would win! I'm up 10 to 1 on this bet!" A nearby Avgi says with exuberance as they buzz around you wildly, holding various pink papers in their many arms.`

Reviewer: ```suggestion
			`	"Hysteria. I knew he would win! I'm up 10 to 1 on this bet!" a nearby Avgi says with exuberance as they buzz around you wildly, holding various pink papers in their many arms.`
```

> 			`	Across the arena you hear a shout from a pink Avgi, "Adoration! I love you Zomp!"`

Reviewer: ```suggestion
			`	Across the arena you hear a shout from a pink Avgi, "Adoration! I love you, Zomp!"`
```
This could probably go either way, but I think it's better with the comma.

Author: I agree it could go either way, but I'll take your suggestion here.
